allennlp_models/structured_prediction/dataset_readers/srl.py

```diff
@@ -133,7 +133,13 @@ def __init__(
         **kwargs,
     ) -> None:
         super().__init__(**kwargs)
-        self._token_indexers = token_indexers or {"tokens": SingleIdTokenIndexer()}
+        if token_indexers is not None:
+            self._token_indexers = token_indexers
+        elif bert_model_name is not None:
+            from allennlp.data.token_indexers import PretrainedTransformerIndexer
+            self._token_indexers = {"tokens": PretrainedTransformerIndexer(bert_model_name, "tokens")}
```

Comment:
Are you sure you want to use "tokens" as the vocab namespace here?

Reply:
I'm assuming it's because of an assumption made in the `srl_bert.py` code.  But, e.g., I just wrote some stuff in the multi-task code that follows the assumption made in `PretrainedTransformerIndexer` that the default namespace is "tags".  It'd be better if we just modified all first-party code to make the same assumption.  I'd be in favor of changing `srl_bert.py` instead of changing the default value here.

Reply:
No, but `SingleId` uses it.

Reply:
I don't understand why the default is `"tags"`. Do you?